### DISCUSSION TITLE
Onion OS for RG35XX And RG35XX Plus

### DISCUSSION THREAD

**Discussion by Coder-Devesh-Agarwal** [OnionUI/Onion#1288] [Ideas]
Can We Have OnionOS support on RG35XX as well as some other handhelds?
I am willing to work, and if you have any insights on how to get started let me know.

**Comment by Knightly-Spartan** [comment]
Sounds like an awesome idea for RG35XX devices, none of their current alternate OSes are as polished as Onion OS on Miyoo

**Comment by vickvohoang** [comment]
Yes Please

**Comment by Datinhkhach** [comment]
+1 for RG35XX Plus

**Comment by Nighted** [comment]
R.I.P. Miyoo. :D

**Comment by jonathan8devs** [comment]
+1 for RG35XX Plus

**Comment by samuel-morrissey** [comment]
Plsssssssss!

**Comment by gamernano1** [comment]
+1 for Rg35xx H. If this was made possible it would be amazing. Happy to back it on patreon.

**Comment by metawops** [comment]
+1 from me. Any news on this? 😳

**Comment by e1ke** [comment]
+1 RG35XX H. 
I dont own one but I would buy one, if there is Onion OS for it. Would be happy to throw money over Patreon as well

**Comment by npaladin2000** [comment]
I would also back this on Patreon and would offer any assistance I can with testing and troubleshooting.

**Comment by Nighted** [comment]
I would pay money for this. Please make it happen. Miyoo fanboys will cry, but they probably do that everyday already.

**Comment by Jo900** [comment]
Agree, I think is a great idea.

**Comment by lumberfoot** [comment]
I would unironically pay for RGB30 support.

**Comment by mont2236** [comment]
After researching Easy Netplay, I realized this is a long-overdue feature. It would be a shame to see it limited only to Miyoo devices, when it has the potential to become a standard across all handheld consoles. Honestly, I would gladly support this through Patreon if it meant seeing it available more broadly.

**Comment by gingerbeardman** [comment]
Would be lovely for 2026/2027
